rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2446,6 +2446,8 @@ def test_upgrade_db_31_to_32(user_data_dir):  # pylint: disable=unused-argument
         assert cursor.execute(
             'SELECT COUNT(*) FROM sqlite_master WHERE type="table" AND name=?', (name,),
         ).fetchone()[0] == 1
+    manual_balances_before = cursor.execute('SELECT asset, label, amount, location, category FROM '
+                                            'manually_tracked_balances;').fetchall()
```

Comment:
The way we split multiple string lines in the code for now is like this:

```suggestion
    manual_balances_before = cursor.execute(
        'SELECT asset, label, amount, location, category FROM '
        'manually_tracked_balances;'
    ).fetchall()
```

Or if it's just a small one and you think it would look fine even if above 99 chars then you can add `  # noqa: E501` at the end of the line which will make flake8 ignore the line warning.